### PR TITLE
Create a team archive command in mmctl

### PR DESCRIPTION
**Pull request by gupsho**
Fixes https://github.com/mattermost/mattermost-server/issues/12963

### CODE REVIEW

File: commands/team.go [hunk 1]
```diff
@@ -222,6 +233,43 @@ func deleteTeam(c client.Client, team *model.Team) (bool, *model.Response) {
...
+	confirmFlag, _ := cmd.Flags().GetBool("confirm")
+	if !confirmFlag {
+		var confirm string
+		fmt.Println("Have you performed a database backup? (YES/NO): ")
+		fmt.Scanln(&confirm)
+
+		if confirm != "YES" {
+			return errors.New("ABORTED: You did not answer YES exactly, in all capitals.")
+		}
+		fmt.Println("Are you sure you want to archive the teams specified?  All data will be deleted? (YES/NO): ")
```
Comment (mgdelacroix) [inline commands/team.go]: As the team is being archived, we can lower down a little the error message:

```suggestion
		fmt.Println("Are you sure you want to archive the specified teams? (YES/NO): ")
```

File: commands/team.go [hunk 2]
```diff
@@ -222,6 +233,43 @@ func deleteTeam(c client.Client, team *model.Team) (bool, *model.Response) {
 	return c.PermanentDeleteTeam(team.Id)
 }
 
+func archiveTeamsCmdF(c client.Client, cmd *cobra.Command, args []string) error {
+	if len(args) < 1 {
```
Comment (mgdelacroix) [inline commands/team.go]: You can replace this check by adding `Args: cobra.MinimumNArgs(1),` to the command definition

File: commands/team.go (hunk 1 above)
Comment (gupsho) [reply commands/team.go]: Done.

File: commands/team.go (hunk 2 above)
Comment (gupsho) [reply commands/team.go]: Done.

**Comment by gupsho** [comment]
> Thanks @gupsho @mgdelacroix - this can be tested post-merge.
> 
> Please remember to add unit tests @gupsho :)

Hi, Could you please help me with the build issue? I see the same error on my machine. I dont believe I added that package.

**Comment by mgdelacroix** [comment]
> > Thanks @gupsho @mgdelacroix - this can be tested post-merge.
> > Please remember to add unit tests @gupsho :)
> 
> Hi, Could you please help me with the build issue? I see the same error on my machine. I dont believe I added that package.

Sure! It was related to a recent change we made in `mattermost-server` to addapt to use gomodules. From this point on, when importing anything from `mattermost-server` we have to use the version suffix. Probably this started to happen when you merged `master`, which already points to this new version.

I've pushed a commit to this PRs branch that should solve the problem and exemplifies this :)

**Pull request merged**